### ISSUE TITLE
AttributeError: 'str' object has no attribute 'get'

### ISSUE THREAD

**Issue by galleon** [whitphx/streamlit-webrtc#287]
When launching the DeepSpeech demo I get this error.

![image](https://user-images.githubusercontent.com/259772/124271603-f2ec6a00-db3d-11eb-875c-7b192692a891.png)

Here is the full error:

```
File "/Users/alleon_g/.pyenv/versions/3.8.6/envs/lewagon/lib/python3.8/site-packages/streamlit/script_runner.py", line 349, in _run_script
    exec(code, module.__dict__)
File "/Users/alleon_g/code/galleon/refactored-octo-tribble/app.py", line 303, in <module>
    main()
File "/Users/alleon_g/code/galleon/refactored-octo-tribble/app.py", line 111, in main
    app_sst_with_video(
File "/Users/alleon_g/code/galleon/refactored-octo-tribble/app.py", line 212, in app_sst_with_video
    webrtc_ctx = webrtc_streamer(
File "/Users/alleon_g/.pyenv/versions/3.8.6/envs/lewagon/lib/python3.8/site-packages/streamlit_webrtc/__init__.py", line 289, in webrtc_streamer
    playing = component_value.get("playing", False)
```

**Comment by trisgelar** [comment]
I got same error message, i changed streamlit and streamlit-webrtc back to older version

streamlit==0.83.0
streamlit-webrtc==0.23.2

**Comment by whitphx** [comment]
Thank you for the report! I will fix it.

**Comment by whitphx** [comment]
It looks like a bug of streamlit==0.84.0.
I created a bug report: https://github.com/streamlit/streamlit/issues/3507

**Comment by whitphx** [comment]
I published a new version of `streamlit-webrtc`, including a hotfix for this problem.
However, I think this is derived from streamlit core and will keep this issue opened until the fundamental error is resolved.

**Comment by whitphx** [comment]
The upstream is fixed in 0.84.1.
